src/panels/lovelace/components/hui-image.ts

```diff
@@ -66,11 +70,37 @@ export class HuiImage extends LitElement {
     if (this.cameraImage && this.cameraView !== "live") {
       this._startUpdateCameraInterval();
```

Comment:
Remove this from `connectedCallback`, it will be handled by the `intersectionObserver`

Reply:
refactored this